.github/workflows/cron.yaml

```diff
@@ -1,78 +1,29 @@
+# Based off https://github.com/rapidsai/cudf/blob/branch-25.02/.github/workflows/pandas-tests.yaml
+name: Test dask-upstream
+
 on:
-  # TODO: change to cron-based schedule one this is working
-  push:
-    branches:
-      - main
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
 
 jobs:
-  test:
-    name: "Test dask and distributed"
-    # TODO: change to appropriate image
-    runs-on: "linux-amd64-gpu-v100-latest-1"
-    container:
-      image: rapidsai/distributed:24.12-cuda11.8.0-devel-ubuntu20.04-py3.12
-      env:
-        NVIDIA_VISIBLE_DEVICES: ${{ env.NVIDIA_VISIBLE_DEVICES }}
-    steps:
-    - name: Checkout ourselves
-      uses: actions/checkout@v4
-      with:
-        path: utils
-    - name: Checkout dask
-      uses: actions/checkout@v4
-      with:
-        repository: dask/dask
-        path: dask
-    - name: Checkout distributed
-      uses: actions/checkout@v4
-      with:
-        repository: dask/distributed
-        path: distributed
-    - name: Run
-      run: |
-        (cd dask; git rev-parse HEAD;
-         cd ../distributed; git rev-parse HEAD) | tee commit-hashes.txt
-    - name: Upload commit hashes
```

Comment:
I've dropped this checking of hashes. IIUC, the idea was to only run the test if dask or distributed chnaged.

There's also the potential that *downstream* changes will break the integration, and since we're using nightly versions of the downstream libraries presumably *something* will change.